packages/@n8n/eslint-config/src/configs/frontend.ts

```diff
@@ -90,6 +90,10 @@ export const frontendConfig = tseslint.config(
 				},
 			],
 			'vue/no-v-html': 'error',
+			'vue/html-indent': ['warn', 'tab'],
```

Comment:
I'd turn this off and let the formatter take care of it

Reply:
Turned this off